Sources/PointFree/Privacy.swift

```diff
@@ -52,7 +59,7 @@ private let privacyPolicy =
 
 private let personalIdentificationInformation = [
   h2(
-    [`class`([Class.pf.type.title3, Class.padding([.mobile: [.top: 2]])])],
+    [`class`([Class.pf.type.responsiveTitle3, Class.padding([.mobile: [.top: 2]])])],
```

Comment:
I created a new responsive type scale system, and these `responsiveTitleX` will fully replace `title3` in the future. just didnt want to do now.